docs/cluster-configuration.md

```diff
@@ -226,6 +226,29 @@ spec:
     progressTimeout: 150
 ```
 
+## Automatic Configuration of Mediated Devices (including vGPUs)
+
+Administrators can provide a list of desired mediated devices (vGPU) types.
```

Comment:
can you please also explain the relation between this and .spec.permittedHostDevices.mediatedDevices ?

Reply:
Sure!